client/state/plugins/installed/actions.js

```diff
@@ -533,6 +538,72 @@ export function fetchSitePlugins( siteId ) {
 	};
 }
 
+/**
+ * Returns an action object to be used in signalling that site's plugins objects have
+ * been received.
+ *
+ * @param  {object[]} sites Sites received
+ * @returns {object}         Action object
+ */
+export function receiveSitesPlugins( sites ) {
+	return {
+		type: SITES_PLUGINS_RECEIVE,
```

Comment:
I find the naming here a bit confusing: are we receiving sites or plugins or plugins of one site, or plugins of all sites?